app/assets/javascripts/browsered/web-payments/helpers.js

```diff
@@ -108,7 +108,7 @@ const getGooglePaymentsConfiguration = () => {
   }
 
   return {
-    environment: window.googlePayMerchantID.length > 0 ? 'PRODUCTION' : 'TEST',
+    environment: (window.googlePayMerchantID.length > 0 && window.gatewayAccountType !== 'test') ? 'PRODUCTION' : 'TEST',
```

Comment:
would it be better to make it `window.gatewayAccountType == 'live'`? in the unlikely event of a new gateway account type...

Reply:
Ah yeah good idea